### DISCUSSION TITLE
Show locked balance as own balance instead of combining with stake

### DISCUSSION THREAD

**Discussion by vixalien** [idena-network/idena-desktop#709] [Ideas]
Currently, the Idena desktop app show 2 wallets: "Main" and "Stake". However the "Stake" wallet contains both the stake balance and locked funds.
 
I think the Idena desktop app should show 3 wallets instead of the current 2. That is "Main", "Stake" and "Locked".

I also think it would look better wiith the main wallet having the old blue background, the stake account holding a slight blue outline and the locked fund normal or with a diagonal line background denoting it's locked.

![image](https://user-images.githubusercontent.com/37999241/134035724-84ecf293-bdfd-40ab-8a3f-1b4f3a512caf.png)

**Comment by aidenaio** [comment]
propose to discussions
